docs/introduction.md

```diff
@@ -4,145 +4,62 @@
 
 ## What is KubeVela?
 
-KubeVela is a framework to help platform builders to easily expose and extend
-Kubernetes capabilities for application management. It is built on top of
-Kubernetes and relieves the pains of both the platform builders and the application
-developers by doing the following:
+For developers, KubeVela itself is an easy-to-use tool that enables them to describe and ship their applications to Kubernetes with minimal effort. Instead of managing a handful Kubernetes YAML files, a simple docker-compose style [Appfile](./docs/developers/devex/appfile.md) is what all they need, following an application-centric workflow that can be easily integrated with any CI/CD pipeline.
 
-- KubeVela enforces a single **application** concept and **ALL** the exposed
-  Kubernetes capabilities serve for the applications only.
-  It adopts the [Open Application Model](https://github.com/oam-dev/spec) (OAM)
-  for its application definition. The conventional Pod and container concepts
-  are completely eliminated.
+For platform builders, KubeVela serve as a framework that empowered platform teams create developer facing yet highly extensible platforms at ease. In detail, KubeVela relieves the pains of building such platforms by doing the following:
+
+- Application Centric. Behind the Appfile, KubeVela enforces the corresponding **application** concept as its main API and **ALL** KubeVela's capabilities serve for the applications' needs only. This is achieved by adopting the [Open Application Model](https://github.com/oam-dev/spec) as the main API for the platform.
  
-- A KubeVela application is composed of various supported components or
-  [traits](https://github.com/oam-dev/spec/blob/master/introduction.md) and their
-  schemas are determined by the platform builders. New capabilities can be added
-  to KubeVela through a CRD registry mechanism. 
-
-- KubeVela provides a rich set of tools to help platform builders to abstract
-  the Kubernetes resource and CRD APIs.
-  For example, a [CUELang](https://github.com/cuelang/cue) based templating tool
-  is used to easily build the contract between the user-facing schemas and the
-  underline Kubernetes Objects. KubeVela provides a set of built-in CUE templates
-  for platform builders to start with.
-
-- Schema changes take effect immediately. Neither recompiliation nor redeployment
-  of KubeVela is required. This makes the process of delivering new capabilities
-  using KubeVela extremely simple.
+- Natively Extensible. An application in KubeVela is composed of various pluggable workload types and operational traits. Capabilities from Kubernetes ecosystem can be added to KubeVela as new workload types or trait through Kubernetes CRD registry mechanism at any time.
```

Comment:
- Natively Extensible -> Extending natively.
- Add a link to `traits` since this term appears in the first time.
- or trait -> or traits

Reply:
Done. The link to "traits" will be added after `docs/concepts.md` is finished, will be sth like this: https://github.com/oam-dev/rudr/blob/master/docs/concepts/traits.md